lib/symengine/basic.rb

```diff
@@ -11,8 +11,13 @@ def free_symbols
     end
     
     def abs
-        return SymEngine::abs(self)
+      SymEngine::abs(self)
+    end
+    def to_proc()
+      if self.free_symbols.count > 1
+        raise ArgumentError, "Too many free symbols! Only 1 allowed. Found #{self.free_symbols.count}."
```

Comment:
@isuruf so, on what the order depends?.. on alphabetical order of symbols?

(And I believe previously it was pointed that symbols order is not guaranteed at all, but I haven't checked it myself)